lib/puppetserver/ca/utils/file_system.rb

```diff
@@ -50,6 +50,17 @@ def self.check_for_existing_files(one_or_more_paths)
           errors
         end
 
+        def self.symlink_to_old_cadir(current_cadir)
+          old_cadir = '/etc/puppetlabs/puppet/ssl/ca'
+          new_cadir = '/etc/puppetlabs/puppetserver/ca'
+          return if current_cadir != new_cadir
+          # Files should have already been moved if we're using the new cadir
```

Comment:
Files won't and shouldn't exist during this.

Reply:
ah yeah, I can make this comment more accurate

Reply:
I think that maybe means we don't need the `rm` right? Since this function is called during initial setup, and will be using the new location?

Reply:
yeah, perhaps.. but seeing how much fun we've had with symlinks in the past, it seems safer to delete it just in case.

Reply:
Heh fair enough. Seems alright I guess.